org.briarproject.Briar.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.briarproject.Briar</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
I don't think this is an acceptable metadata license.

CC0-1.0 is preferred usually.